### ISSUE TITLE
Unknown debug messages from Editra?

### ISSUE THREAD

**Issue by robmcmullen** [robmcmullen/peppy#591]
_[Trac time 20090222 200455Z]_ The following messages appear at startup when using OS X:

```
[Debug] 13:09:58: Unrecognized accel key 'numpad_add', accel string ignored.
[Debug] 13:09:58: Unrecognized accel key 'numpad_add', accel string ignored.
[Debug] 13:09:58: Unrecognized accel key 'numpad_subtract', accel string ignored.
[Debug] 13:09:58: Unrecognized accel key 'numpad_subtract', accel string ignored.
```

Those aren't normal peppy debug strings -- it might be from the Editra styling code?

**Comment by robmcmullen** [comment]
_[Trac time 20090416 162148Z]_ Nope, they are from wx itself -- apparently you can't put any NUMPAD_ keybindings into menu accelerators.

**Comment by robmcmullen** [comment]
_[Trac time 20090416 183408Z]_ 8b410e6e6ae0aac72c77bd2eb59f31734db330c4 Fixed #591: workaround the debug error message by only using the NUMPAD menu accelerators for emacs mode
